### ISSUE TITLE
[FP] Placing snow in flowing water causes interact.liquid to false

### ISSUE THREAD

**Issue by AMDBartek** [jiangdashao/Matrix-Issues#2109]
## Troubleshooting Information
`Change - [ ] to - [X] to check the checkboxes below.`
- [X] Matrix and ProtocolLib are up-to-date
- [X] Matrix is running on a 1.8, 1.12, 1.13, 1.14, 1.15 server
- [X] The issue happens on default config.yml and checks.yml
- [X] I've tested if the issue happens on default config

## Issue Information
**Server version**: 1.12.2-R0.1-SNAPSHOT git-Paper-1618 (MC: 1.12.2)
**Verbose messages**: 
```
(06:53:03) YouAreAYeet failed Interact check | interact with liquid (at 3564.0,72.0,1907.0) (+1) | vl:1 ping:8 tps:18.78 lp:10 component: interact.liquid
(06:53:43) YouAreAYeet failed Interact check | interact with liquid (at 3564.0,72.0,1907.0) (+1) | vl:1 ping:4 tps:19.78 lp:13 component: interact.liquid
(06:54:00) YouAreAYeet failed Interact check | interact with liquid (at 3564.0,72.0,1907.0) (+1) | vl:2 ping:3 tps:19.69 lp:20 component: interact.liquid
(06:54:01) YouAreAYeet failed Interact check | interact with liquid (at 3564.0,72.0,1907.0) (+1) | vl:3 ping:3 tps:19.69 lp:14 component: interact.liquid
(06:58:02) YouAreAYeet failed Interact check | interact with liquid (at 3564.0,72.0,1907.0) (+1) | vl:1 ping:1 tps:19.85 lp:1 component: interact.liquid
(06:58:02) YouAreAYeet failed Interact check | interact with liquid (at 3564.0,72.0,1907.0) (+1) | vl:2 ping:1 tps:19.85 lp:1 component: interact.liquid
(06:58:24) YouAreAYeet failed Interact check | interact with liquid (at 3564.0,72.0,1908.0) (+1) | vl:1 ping:8 tps:19.92 lp:0 component: interact.liquid
```
**How/when does this happen**: When placing snow in flowing water.
**Video of false positive**: https://streamable.com/dheq18
**Other information**: Sometimes it falses a little bit but sometimes it falses to the point where the player is kicked

## Configuration Files
**Link to checks.yml file**: Default

**Comment by jiangdashao** [comment]
not important

**Comment by AMDBartek** [comment]
@jiangdashao will this be fixed in 5.3.2 or no?

**Comment by jiangdashao** [comment]
no
